packages/expo-modules-core/src/NativeViewManagerAdapter.native.tsx

```diff
@@ -2,7 +2,7 @@
 
 'use client';
 
-import React from 'react';
+import { type Component, PureComponent, createRef, ComponentType } from 'react';
```

Comment:
```suggestion
import { type Component, type ComponentType, PureComponent, createRef } from 'react';
```

`ComponentType` is also a type